src/components/Footer/index.js

```diff
@@ -37,62 +39,109 @@ export const OfficeStyled = styled(Row)`
   overflow: hidden;
 `
 
-const Footer = () => {
-  return (
-    <GreyFooter>
-      <Padding top={{ smallPhone: 3, tablet: 4 }}>
+const DesktopGDPR = styled.div`
+  visibility: hidden;
+  position: absolute;
+  right: 0;
+  color: white;
+
+  a {
+    margin-left: 24px;
+  }
+
+  ${breakpoint('desktop')`
+    visibility: visible;
+  `}
+`
+
+const MobileGDPR = styled.div`
+  color: white;
+
+  a {
+    display: inline-block;
+    margin-right: 24px;
+    margin-bottom: 24px;
+  }
+
+  ${breakpoint('desktop')`
+    visibility: hidden;
+  `}
+`
+
+const LinkUnderline = styled(Link)`
+  text-decoration: underline;
+`
+
+const gdprLinksElement = gdpr.map(link => (
+  <LinkUnderline key={link.text} to={link.to}>
+    {link.text}
+  </LinkUnderline>
+))
+
+const Footer = () => (
+  <GreyFooter>
+    <Padding top={{ smallPhone: 3, tablet: 4 }}>
+      <Grid>
+        <Row>
+          <Col width={1}>
+            <Padding bottom={3}>
+              <SectionTitle reverse>Find us</SectionTitle>
+            </Padding>
+          </Col>
+        </Row>
+        <Padding bottom={{ smallPhone: 3.5, tablet: 5 }}>
+          <OfficeStyled>
+            <Locations>
+              {data =>
+                data.map(location => {
+                  const streetAddress = location.node.streetAddress.streetAddress.split(
+                    '\n'
+                  )
+
+                  return (
+                    <Office
+                      key={location.node.id}
+                      {...location.node}
+                      streetAddress={streetAddress}
+                    />
+                  )
+                })
+              }
+            </Locations>
+          </OfficeStyled>
+        </Padding>
+      </Grid>
+    </Padding>
+    <FollowUs>
```

Comment:
Given that this black banner now contains GDPR links, as well as social links, the name `FollowUs` is probably no longer appropriate for the black banner.

Perhaps you want to change this to `BlackBanner` or `BottomBanner` or `FooterEnd`, or something else that's better?

Reply:
Good idea. I mixed your solution and it's now `FooterBanner`